catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/MetacardFactory.java

```diff
@@ -56,50 +59,38 @@ Metacard generateMetacard(String mimeTypeRaw, String id, String fileName, Path t
             throws MetacardCreationException, MimeTypeParseException {
 
         Metacard generatedMetacard = null;
-        InputTransformer transformer = null;
-        StringBuilder causeMessage = new StringBuilder("Could not create metacard with mimeType ");
-        try {
-            MimeType mimeType = new MimeType(mimeTypeRaw);
-
-            List<InputTransformer> listOfCandidates = mimeTypeToTransformerMapper.findMatches(
-                    InputTransformer.class,
-                    mimeType);
-
-            LOGGER.debug("List of matches for mimeType [{}]: {}", mimeType, listOfCandidates);
-
-            for (InputTransformer candidate : listOfCandidates) {
-                transformer = candidate;
-
-                try (InputStream transformerStream = com.google.common.io.Files.asByteSource(
-                        tmpContentPath.toFile())
-                        .openStream()) {
-                    generatedMetacard = transformer.transform(transformerStream);
-                }
-                if (generatedMetacard != null) {
-                    break;
-                }
+
+        MimeType mimeType = new MimeType(mimeTypeRaw);
+
+        List<InputTransformer> listOfCandidates = mimeTypeToTransformerMapper.findMatches(
+                InputTransformer.class,
+                mimeType);
+        List<String> stackTraceList = new ArrayList<>();
+
+        LOGGER.debug("List of matches for mimeType [{}]: {}", mimeType, listOfCandidates);
+
+        for (InputTransformer candidate : listOfCandidates) {
```

Comment:
After looking at this, it's not easily feasible to remove this logic (the MetacardFactory methods are protected, not published as a service, etc).  I've refactored the code to match the code changes in the MetacardFactory.